ftw/iframeblock/upgrades/20191217091516_add_resources_for_plone5_in_production/registry.xml

```diff
@@ -0,0 +1,19 @@
+<registry>
+
+    <record name="plone.types_not_searched" interface="Products.CMFPlone.interfaces.controlpanel.ISearchSchema" field="types_not_searched">
+        <value>
+            <element>ftw.iframeblock.IFrameBlock</element>
+        </value>
+    </record>
+
+    <records prefix="plone.bundles/ftw-iframeblock-resources" interface='Products.CMFPlone.interfaces.IBundleRegistry'>
+        <value key="enabled">True</value>
+        <value key="depends">plone</value>
+        <value key="stub_js_modules">jquery</value>
+        <value key="compile">False</value>
+        <value key="jscompilation">++resource++ftw.iframeblock/iframeblock-compiled.js</value>
+        <value key="last_compilation">2019-12-10 11:00:00</value>
```

Comment:
Why two upgrade steps? The other one does the same.

Reply:
I didn't notice the existing. I removed it in 8080a2f